config/test_dtm_config.yaml

```diff
@@ -12,3 +16,6 @@ test_41234:
 test_41235:
   nclients: 10
   nsamples: 10
+
+test_42246:
+  num_loop: 300
```

Comment:
We can keep this 100.

Reply:
As we are planning to restart process 5 times, better to keep this to 300 buckets for safe side, so that some deletes happen after process restarts as well.